GeeDollaHolla/canari-swipe
# [HOLD]: Refactor file structure

I went ahead and restructured the file structure so that it would be easier to test and maintain.  The naming structing (having folders with an `index.js` in it) is opinionated so let me know if you'd rather not. 

## Deprecations
* I removed the constructor in favor of `class-transform-properties`
* `babel-preset-2015` references were removed in favor of the already-installed `babel-preset-env`

## To verify changes

I went ahead and ran `npm run demo-build` to verify the changes and everything ran successfully, however, I ran `npm run build` and the index.js file created could not be found.  I'll keep this PR open on my end so I can push changes as needed.  

If accepted, I'd be more than happy to add tests to showcase coverage.
